### ISSUE TITLE
.git is added to the package

### ISSUE THREAD

**Issue by lepouletsuisse** [kefranabg/readme-md-generator#97]
Hello!

I juste updated readme-md-generator to version 0.6.0 and it seems there is the .git directory that has been added to the package. That implies that when when we try to do a `npm install` when readme-md-generator is already installed, we got the following error:

> npm ERR! path node_modules/readme-md-generator  
> npm ERR! code EISGIT  
> npm ERR! git node_modules/readme-md-generator: Appears to be a git repo or submodule.  
> npm ERR! git node_modules/readme-md-generator  
> npm ERR! git Refusing to remove it. Update manually,  
> npm ERR! git or move it out of the way first.  

To reproduce it, simply install readme-md-generator@0.6.0, install it and than try to do an other `npm install`

Have a good day!

Le Poulet Suisse

**Comment by kefranabg** [comment]
Hi @lepouletsuisse,

I just tested and it's working as expected for me.
Can you tell me what command you are using ?

To get it work, juste use a recent version of npm and do the following :

```
npx readme-md-generator
```
or

```
npm i -g readme-md-generator
readme
```

**Comment by lepouletsuisse** [comment]
Hi @kefranabg ,
In fact i'm doing a `npm i --save readme-md-generator` and than any `npm i ...` command got the error i showed in my first comment. 
I tried to install it globally and it seems to work. Anyway, i think this is still a bug if one wants to install the package in the project only. 
npm version: 6.9.0
Node version: 10.16.0
readme-md-generator version: 0.6.0
Here it is a screenshot that show the .git folder inside the package when we install it. The .git package shouldn't be publish with the package i think.
![Screenshot from 2019-07-18 08-16-27](https://user-images.githubusercontent.com/8804334/61433900-d1841a80-a934-11e9-85b7-1e299052aea2.png)

**Comment by kefranabg** [comment]
@lepouletsuisse, the .git is added in npm repository indeed...
However it's not supposed to be possible.

I made some research and find out that it's a bug  from `npm@6.9.1`, the version that I used to publish readme-md-generator.

I'll use another version to fix this issue.
Thanks for reporting 👍

**Comment by lepouletsuisse** [comment]
Oh ok, Thank you for your work :smiley:  :+1:

**Comment by kefranabg** [comment]
Fixed in [0.6.1](https://github.com/kefranabg/readme-md-generator/commit/f940f130efef5ffdeb3f278e7435570c95363acb) 👍
